lib/content-services/src/lib/site-dropdown/sites-dropdown.component.html

```diff
@@ -1,5 +1,6 @@
 <div id="site-dropdown-container" class="adf-site-dropdown-container">
     <mat-form-field>
+        <mat-label>{{ 'NODE_SELECTOR.SELECT_LOCATION' | translate }}</mat-label>
```

Comment:
If we're adding label here please also ask Shane to decide if we need both placeholder and label here or just label

Reply:
yeah we need both label and placeholder, i confirmed with shane.